test/internals/routes.js

```diff
@@ -178,12 +178,23 @@ describe('Internals - routes', () => {
       }
     })
 
+    it('should return the root url of the app if not passed a path', async () => {
+      const mock = {
+        path: undefined
+      }
+
+      const output = fullyQualifiedLocalPath(mock.path)
+
+      expect(output).to.be.a.string()
+      expect(output).to.equal(mockAppDomain)
+    })
+
     it('should return a fully qualified url including the passed path', async () => {
       const mock = {
         path: 'just-a-path'
       }
 
-      const output = await fullyQualifiedLocalPath(mock.path)
+      const output = fullyQualifiedLocalPath(mock.path)
```

Comment:
Using this object to declare the mock path is inconsistent with the test that follows - they should either both be declared with a simple mockPath string or with an object as here. (Mark)